src/components/ToplineBar.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { COLORS } from '@styles/themes';
+
+const styles = StyleSheet.create({
+  view: {
+    backgroundColor: COLORS.errorBgColor,
```

Comment:
Why did you use this as default bg color?

Reply:
I used this color just to pick a default color instead of use a custom one. In addition, this is the default color for the `OfflineBar`.